gnosis/eth/clients/etherscan_client.py

```diff
@@ -31,6 +31,8 @@ class EtherscanClient:
         EthereumNetwork.BINANCE: "https://bscscan.com",
         EthereumNetwork.MATIC: "https://polygonscan.com",
         EthereumNetwork.OPTIMISTIC: "https://optimistic.etherscan.io",
+        EthereumNetwork.BOBA_RINKEBY: "https://blockexplorer.rinkeby.boba.network/",
```

Comment:
This is not etherscan, it should be removed from here

Reply:
ah, I completely missed the class name.